packages/flutter_tools/templates/ffi_plugin/ios.tmpl/Classes/dart_api_dl.c

```diff
@@ -0,0 +1,2 @@
+// Relative import to be able to reuse the c sources.
+#include "../../src/dart-sdk/include/dart_api_dl.c"
```

Comment:
comment from api design review: before we make it even easier to link to the Dart API headers, we should have a conversation about the long term stability guarantees of these headers.

Reply:
cc @chinmaygarde @dnfield

Reply:
I'll push this out to a follow up PR.